lib/galaxy/tool_util/deps/conda_util.py

```diff
@@ -494,42 +513,23 @@ def cleanup_failed_install(conda_target, conda_context=None):
     cleanup_failed_install_of_environment(conda_target.install_environment, conda_context=conda_context)
 
 
-def best_search_result(conda_target, conda_context, channels_override=None, offline=False, platform=None):
```

Comment:
It looks like this breaks Planemo?

Reply:
@jmchilton Sorry, you are right. I don't particularly like the `channels_override` argument, so I'll propose 2 possible fixes:
1) Restore the function argument: https://gist.github.com/nsoranzo/089ef056d233a4d839716b77117fa120
2) Update Planemo to not use it: https://gist.github.com/nsoranzo/bcffb9cfffba529a92b8e5d02f44c808

Which one do you prefer?

Reply:
https://github.com/galaxyproject/planemo/pull/1224